### ISSUE TITLE
Bug: onBtnPressed ignores middle mouse button clicks

### ISSUE THREAD

**Issue by Litoprobka** [fjvallarino/monomer#63]
A `box` with `onBtnPressed` / `onBtnReleased` completely ignores clicks on the middle mouse button.

Here's a small isolated example.
<details>
<summary> example </summary>

```haskell
{-# LANGUAGE OverloadedStrings #-}
{-# LANGUAGE TemplateHaskell #-}

module Main where

import Control.Lens
import Data.Maybe
import Data.Text (Text, pack)
import Monomer
import TextShow

import qualified Monomer.Lens as L

newtype AppModel = AppModel {
  _lastClick :: Text
} deriving (Eq, Show)

data AppEvent
  = AppInit
  | LastClick Text
  deriving (Eq, Show)

makeLenses 'AppModel

buildUI
  :: WidgetEnv AppModel AppEvent
  -> AppModel
  -> WidgetNode AppModel AppEvent
buildUI wenv model = widgetTree where
  widgetTree =
      box_ [expandContent, onBtnPressed handleClick]
        (label (model ^. lastClick) `styleBasic` [textCenter])

handleClick b times = LastClick $ pack (show b) <> " pressed " <> showt times <> " times"

handleEvent
  :: WidgetEnv AppModel AppEvent
  -> WidgetNode AppModel AppEvent
  -> AppModel
  -> AppEvent
  -> [AppEventResponse AppModel AppEvent]
handleEvent wenv node model evt = case evt of
  AppInit -> []
  LastClick t -> [Model (model & lastClick .~ t)]

main :: IO ()
main = do
  startApp model handleEvent buildUI config
  where
    config = [
      appWindowTitle "Hello world",
      appTheme darkTheme,
      appFontDef "Regular" "./assets/fonts/Roboto-Regular.ttf",
      appInitEvent AppInit
      ]
    model = AppModel "No button presses yet"
```
</details>

Tested on:

- Void Linux PC (mouse)
- Void Linux laptop (touchpad) 
-  Windows 10 laptop (mouse)  

with Monomer 1.2.0.0

**Comment by fjvallarino** [comment]
Hi @Litoprobka!

I had missed converting middle button actions from SDL2 events to Monomer events. I pushed a new branch (the PR is linked to this issue). To use it, you will need to update your `stack.yaml`:

```yaml
- git: https://github.com/fjvallarino/monomer.git
  commit: d34cd780c8cc0244b40da09e69dcbff316e75555
```

Thanks for reporting this!

**Comment by Litoprobka** [comment]
Thank you for replying so fast!

**Comment by fjvallarino** [comment]
Since I merged to main and deleted the branch, the commit hash mentioned previously does not exist anymore. Below is the new one from main (which will never be removed). This change will be included in 1.3, which ideally will be released in 1-2 weeks.

```yaml
- git: https://github.com/fjvallarino/monomer.git
  commit: 4e0f8fadab633718ecdcfa7322e66e9ec58a7ee6
```
